addons/sourcemod/translations/freak_fortress_2.phrases.txt

```diff
@@ -4,6 +4,14 @@
 	{
 		"en"	"Freak Fortress 2 is currently disabled. More players are required."
 	}
+	"ff2_group"
+	{
+		"en"	"VS Saxton Hale/Freak Fortress 2 group: {olive}http://steamcommunity.com/groups/vssaxtonhale{default}"
+	}
+	"ff2_devs"
+	{
+		"en"	"{default} === Freak Fortress 2 v%s (based on VS Saxton Hale Mode by {olive}RainBolt Dash{default}, {olive}FlaminSarge{default}, and {blue}Chdata{default}) === "
```

Comment:
Above this line there needs to be a `"#format" "{1:s}", and `v%s` needs to be changed to `v{1}`